proto/src/proto.rs

```diff
@@ -260,14 +260,51 @@ pub enum LdapOp {
 
 #[derive(Clone, PartialEq)]
 pub enum LdapBindCred {
-    Simple(String), // Sasl
+    Simple(String),        // Sasl
```

Comment:
Outdated comment, //Sasl was there as a reminder for future that "this is where it goes".